Dockerfile

```diff
@@ -21,6 +21,10 @@ RUN dotnet publish Jellyfin.Server --disable-parallel --configuration Release --
 FROM jellyfin/ffmpeg:${FFMPEG_VERSION} as ffmpeg
```

Comment:
Should we use this or install it as a package like the other dockerfiles for consistency? anymore additional changes i want to do post 10.5-rc

Reply:
Like this is ideal, since it makes rebuilding when the ffmpeg container changes much easier.

Reply:
How should we handle it for arm?

Reply:
Ideally, the same way with Docker images built for ARM. Would just need to figure that out first. Like most of jellyfin-ffmpeg, more OS support in its build would be ideal.